### PR TITLE
Add changes to file

### PR DESCRIPTION
**Pull request by obed300**
In this project, I restructured my Awesome books app code I did earlier to make it more organized by using modules. I also practiced ES6 syntax.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -64,5 +63,4 @@ window.onload = () => {
     const dt =DateTime.now(); 
   date.innerHTML = dt.toLocaleString(DateTime.DATETIME_FULL);
```
Comment (TedLivist) [inline index.js:65]: - [ ] In this file kindly refactor the following functions to use the arrow syntax based on the ES6 requirement.

  image

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @obed300,

Good job so far!
✔️ Use of luxon package for date and time
✔️ Use of module imports and exports
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (obed300) [reply index.js:65]: Sure. I will do that

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @obed300,

✔️ Use of ES6 syntax for methods

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**